src/config/paths.js

```diff
@@ -14,6 +14,9 @@ export const EXECUTIONS_PATH = '/executions';
 export const VISUALIZATIONS_PATH = '/visualizations';
 export const SCHEMAS_PATH = '/schemas';
 export const SCHEMA_PATH = '/schema/:id';
+export const PIPELINES_PATH = '/pipelines';
+export const EDIT_PIPELINE_PATH = '/edit-pipeline/:id';
```

Comment:
You already have `buildEditPipelinePath` so you don't need the line above.

Reply:
`EDIT_PIPELINE_PATH` is used for the Route path in `App.js`,  whereas `buildEditPipelinePath` is used for building a custom path for the `edit pipeline` page. I just followed the same logic as used for the `schema` or the `dataset` in `path.js`.